docker/basics/step8_remove/verify.sh

```diff
@@ -1,3 +1,36 @@
 #!/bin/bash
 
-#docker run -it --rm --name hello-world alpine:latest "echo Hello World"
\ No newline at end of file
+#docker run -it --rm --name hello-world alpine:latest "echo Hello World"
+
+if ! history | grep -q "docker ps"; then
```

Comment:
I think this check is not needed here. Also, it might succeed anyway if the student typed it at any point earlier in the course.

Reply:
Maybe we could omit an check here at all? For now we will check for an existing container named "nginx_fluct"